exercises/C-comparison-operators/exercise.js

```diff
@@ -5,16 +5,17 @@
   The variables should have values that match the expected results.
 */
 
+
 let studentCount = 16;
 let mentorCount = 9;
-let moreStudentsThanMentors; // finish this statement
+let moreStudentsThanMentors = true; // finish this statement
```

Comment:
While this works for this case, here you can write your code as an expression. For example:

`let moreStudentsThanMentors = studentCount > mentorCount;`